scr/main/java/bittercode/model/User.java

```diff
@@ -0,0 +1,88 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class User implements Serializable {
+
+    private String emailId;
+    private String password;
+    private String firstName;
+    private String lastName;
+    private Long phone;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Phone number stored as Long type</b></div>
<div id="fix">

Consider using `String` instead of `Long` for phone numbers to preserve leading zeros and handle international formats. Phone numbers are not used for arithmetic operations.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12,1 +12,1 @@
 -    private Long phone;
 +    private String phone;
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/1#issuecomment-2514333017>#855223</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged